bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -126,8 +126,12 @@ save_fig_with_metadata(time_str, time_file.storage_path, **kwds)
 
 # Get segments and find the data locations
 sci_seg_name = 'science'
+science_tags = []
+if 'science' in workflow.cp.get_subsections('workflow-segments'):
+    science_tags = ['science']
```

Comment:
I'd just remove these 3 lines and use `tags=['science']` when calling `wf.get_segments_file`. If the section doesn't exist, it just gets ignored.